packages/ui/src/ImportScreen.tsx

```diff
@@ -145,6 +150,21 @@ export default function ImportScreen(): JSX.Element {
     return false;
   }
 
+  function getTagSelectOptions(network: Network): TagSelectOption[] {
```

Comment:
why creating a function? it is been called just once so instead of this it should be constants

Reply:
True. Fixed!